<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIRenameNumber</class>
 <widget class="QWidget" name="UIRenameNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="m_pqgbNumberingIncrement">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Increment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="IComQLineEdit" name="m_pqleNumberingStartNum">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_pqlblNumberingIncrement">
        <property name="text">
         <string>Increment</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="IComQLineEdit" name="m_pqleNumberingIncrement">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_pqlblNumberingStartNum">
        <property name="text">
         <string>Start Number</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="m_pqgbNumberingZeroFill">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Zero Fill</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="m_pqrbNumberingZeroFillAuto">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Automatic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="m_pqrbNumberingZeroFillSpec">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Specified</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="IComQLineEdit" name="m_pqleNumberingZeroFill">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_pqgbNumberingType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Numbering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="m_pqrbNumberingNoNumber">
        <property name="text">
         <string>No Numbering</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pqrbNumberingAfterName">
        <property name="text">
         <string>After Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pqrbNumberingBeforeName">
        <property name="text">
         <string>Before Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="m_pqrbNumberingAtPos">
          <property name="text">
           <string>At Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="IComQLineEdit" name="m_pqleNumberingAtPos">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IComQLineEdit</class>
   <extends>QLineEdit</extends>
   <header>IComQLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_pqrbNumberingNoNumber</tabstop>
  <tabstop>m_pqrbNumberingAfterName</tabstop>
  <tabstop>m_pqrbNumberingBeforeName</tabstop>
  <tabstop>m_pqrbNumberingAtPos</tabstop>
  <tabstop>m_pqleNumberingAtPos</tabstop>
  <tabstop>m_pqleNumberingStartNum</tabstop>
  <tabstop>m_pqleNumberingIncrement</tabstop>
  <tabstop>m_pqrbNumberingZeroFillAuto</tabstop>
  <tabstop>m_pqrbNumberingZeroFillSpec</tabstop>
  <tabstop>m_pqleNumberingZeroFill</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pqrbNumberingAtPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleNumberingAtPos</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbNumberingAtPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleNumberingAtPos</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbNumberingNoNumber</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleNumberingStartNum</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbNumberingNoNumber</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqrbNumberingZeroFillAuto</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbNumberingNoNumber</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleNumberingIncrement</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbNumberingNoNumber</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqrbNumberingZeroFillSpec</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
